cmd/prometheus/config.go

```diff
@@ -334,7 +334,7 @@ func parsePrometheusURL() error {
 	}
 
 	if ok := govalidator.IsURL(cfg.prometheusURL); !ok {
-		return fmt.Errorf("invalid Prometheus URL: %s", cfg.prometheusURL)
+		return fmt.Errorf("invalid Prometheus URL: %s, allowed characters: a-z 0-9 .", cfg.prometheusURL)
```

Comment:
Can you remove the `.` at the end? Errors are not supposed to end with a punctuation.